src/actions/signUpAction.ts

```diff
@@ -0,0 +1,9 @@
+import { AnyAction } from 'redux'
+import { ActionSignup } from '../models/actionModels/signup'
+
+export function autoPopulateEmail(email: string): ActionSignup | AnyAction {
```

Comment:
Action function name can be append with "Action" for better readability. i.e. can be rename this to `autoPopulateEmailAction`